README.md

```diff
@@ -19,9 +19,40 @@ Content is released under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.
 
 ## Acknowledgments
```

Comment:
README.md

Reply:
@@ -19,9 +19,40 @@ Content is released under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.

 ## Acknowledgments